packages/mattermost-redux/src/actions/users.ts

```diff
@@ -170,6 +188,74 @@ export function loadMe(): ActionFunc {
     };
 }
 
+export function loadMeGQL(): ActionFunc {
+    return async (dispatch: DispatchFunc, getState: GetStateFunc) => {
+        const state = getState();
+
+        const deviceId = state.entities.general.deviceToken;
```

Comment:
Not something to change right now, but I don't think we actually use the deviceToken any more since it was part of the mobile app. I've filed a separate ticket to remove the usage of it as well as some other unused fields I found in `state.entities.general`

Reply:
thank you would you like assign me that ticket?